ruby/ql/lib/codeql/ruby/security/ConditionalBypassCustomizations.qll

```diff
@@ -0,0 +1,58 @@
+/**
+ * Provides default sources, sinks and sanitizers for reasoning about
+ * bypass of sensitive action guards, as well as extension points for
+ * adding your own.
+ */
+
+private import codeql.ruby.CFG
+private import codeql.ruby.DataFlow
+private import codeql.ruby.controlflow.BasicBlocks
+private import codeql.ruby.dataflow.RemoteFlowSources
+private import codeql.ruby.security.SensitiveActions
+
+module ConditionalBypass {
```

Comment:
Missing a QLDoc.

Reply:
We're a little bit inconsistent about this, but I notice that most of the other `*Customizations.qll` files don't have a QLDoc on the module they define.

Reply:
In this case we're failing a PR check because of the missing QLDoc, it looks like that check only applies to changed files.

Reply:
Whoops, I didn't spot that. I'll fix it.